tests/e2e/fixtures/cairo1_project/tests/test_a.cairo

```diff
@@ -0,0 +1,12 @@
+#[test]
+fn test_assert_trueA() {
+    let x = 1;
```

Comment:
This doesn't check anything, shouldn't we `assert`, especially since the method name itself says we do so? Or if `assert` is not an option, I'd rather rename the function and leave it empty 🤷 Better do less than do confusing stuff if you ask me.

Reply:
Yup thats an important detail